cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -11,40 +11,50 @@ Wikipedia Pipeline
 
 A Cask Data Application Platform (CDAP) example demonstrating a typical data processing pipeline using CDAP Workflows.
 
+
 Overview
 ========
-
 This example demonstrates a CDAP application performing analysis on Wikipedia data using MapReduce and Spark programs
 running within a CDAP Workflow: *WikipediaPipelineWorkflow*.
 
 This example can be run in both online and offline modes.
 
-- In the online mode, the MapReduceProgram *WikipediaDataDownloader* reads the stream *pageTitleStream*, each event of which
-  is an element from the output of the
-  `Facebook "Likes" API <https://developers.facebook.com/docs/graph-api/reference/v2.4/object/likes>`__.
-  For each event, it tries to download Wikipedia data for the page using the
-  `MediaWiki Wikipedia API <https://www.mediawiki.org/wiki/API:Main_page>`__. It stores the downloaded data in the
-  KeyValueTable dataset *wikiData*.
-  
-- In the offline mode, it expects Wikipedia data formatted per the output of the MediaWiki API in the stream *wikiStream*.
-  The MapReduce program *wikiDataToDataset* consumes this stream and stores it in the same KeyValueTable dataset
-  *wikiData*. Data can be uploaded to the *wikiStream* using the CDAP CLI.
+- In the **online mode**, the MapReduceProgram *WikipediaDataDownloader* reads the stream
+  *pageTitleStream*, each event of which is an element from the output of the `Facebook
+  "Likes" API
+  <https://developers.facebook.com/docs/graph-api/reference/v2.4/object/likes>`__. For
+  each event, it tries to download Wikipedia data for the page using the `MediaWiki
+  Wikipedia API <https://www.mediawiki.org/wiki/API:Main_page>`__. It stores the
+  downloaded data in the ``KeyValueTable`` dataset *wikiData*.
+
+..
+
+- In the **offline mode**, it expects Wikipedia data formatted following the output of the
+  MediaWiki API in the stream *wikiStream*. The MapReduce program *wikiDataToDataset*
+  consumes this stream and stores it in the same ``KeyValueTable`` dataset *wikiData*.
+  Data can be uploaded to the *wikiStream* using the CDAP CLI.
 
 Once raw Wikipedia data is available from using either the online or offline modes, the
 *WikipediaPipelineWorkflow* runs a MapReduce program *WikiContentValidatorAndNormalizer* that filters bad records from
 the raw data, as well as normalizes it by converting the wikitext-formatted data to plain text. It then stores the
-output in another KeyValueTable dataset *normalized*.
+output in another ``KeyValueTable`` dataset called *normalized*.
+
+The *WikipediaPipelineWorkflow* then contains a fork, with two branches:
+
+- One branch runs the Apache Spark program *SparkWikipediaAnalyzer*. This program consumes
+  normalized data and runs topic modeling on it using the Latent Dirichlet Allocation (LDA)

```

Comment:
perhaps a link for LDA? https://en.wikipedia.org/wiki/Latent_Dirichlet_allocation